lib/full_text_search/mapper.rb

```diff
@@ -90,8 +90,12 @@ def initialize(mapper, record)
       @record = record
     end
 
-    def find_fts_target
-      Target.find_or_initialize_by(fts_target_keys)
+    def find_fts_target(initialize: true)
```

Comment:
When do we need this?

Reply:
We use `initialize: false` when updating `JournalMapper` in `IssueMapper`.
To avoid https://github.com/clear-code/redmine_full_text_search/commit/2507b826f4a64ba68bb3b0de0d250c46f4163646 issue, we don't initialize when updating `JournalMapper` in `IssueMapper`.